gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
-#Sun May 28 17:31:34 MSK 2017
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
-distributionSha256Sum=71a787faed83c4ef21e8464cc8452b941b5fcd575043aa29d39d15d879be89f7
+#Fri Jan 26 17:08:07 PST 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionSha256Sum=71a787faed83c4ef21e8464cc8452b941b5fcd575043aa29d39d15d879be89f7

```

Comment:
This sha256 sum is wrong.

Reply:
Good catch; I just had android Studio make this change for me and did not realize it failed to properly update the checksum. Will make sure the updated version does contain a valid one.